### ISSUE TITLE
Formie + Craft 4 can't delete submissions

### ISSUE THREAD

**Issue by RyanRoberts** [verbb/formie#999]
### Describe the bug

Hi, when I try to delete Formie submissions I get the following error both locally and on a staging VPS:

```
yii\base\UnknownMethodException: Calling unknown method: verbb\formie\elements\Form::getFields() in /var/www/html/vendor/yiisoft/yii2/base/Component.php:300
Stack trace:
#0 /var/www/html/vendor/craftcms/cms/src/base/Element.php(2026): yii\base\Component->__call('getFields', Array)
#1 /var/www/html/vendor/verbb/formie/src/elements/Submission.php(1044): craft\base\Element->__call('getFields', Array)
#2 /var/www/html/vendor/craftcms/cms/src/services/Elements.php(1672): verbb\formie\elements\Submission->beforeDelete()
#3 /var/www/html/vendor/craftcms/cms/src/elements/actions/Delete.php(189): craft\services\Elements->deleteElement(Object(verbb\formie\elements\Submission))
#4 /var/www/html/vendor/craftcms/cms/src/controllers/ElementIndexesController.php(240): craft\elements\actions\Delete->performAction(Object(verbb\formie\elements\db\SubmissionQuery))
#5 [internal function]: craft\controllers\ElementIndexesController->actionPerformAction()
#6 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#7 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#8 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('perform-action', Array)
#9 /var/www/html/vendor/craftcms/cms/src/web/Application.php(301): yii\base\Module->runAction('element-indexes...', Array)
#10 /var/www/html/vendor/craftcms/cms/src/web/Application.php(625): craft\web\Application->runAction('element-indexes...', Array)
#11 /var/www/html/vendor/craftcms/cms/src/web/Application.php(280): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#12 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#13 /var/www/html/public/index.php(22): yii\base\Application->run()
#14 {main}
```

Also:

```
POST https://gch.ddev.site/admin/actions/element-indexes/perform-action?site=default&v=1658691596719 at 2022-07-24 08:39:56 pm by 172.23.0.1
```

### Steps to reproduce

1. Go to Formie
2. Go to submissions
3. Try to delete a submission from the list, or
4. Try to delete a submission from within the submission

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

4.1.4.1

### Plugin version

2.0.3

### Multi-site?

Yes

**Comment by engram-design** [comment]
Should be fixed in [2.0.4](https://github.com/verbb/formie/releases/tag/2.0.4)

**Comment by RyanRoberts** [comment]
> Should be fixed in [2.0.4](https://github.com/verbb/formie/releases/tag/2.0.4)

Thank you 👍
